### PR TITLE
Message page ui

### PR DESCRIPTION
**Pull request by Kartihan-sri**
Clean up and message page ui

### CODE REVIEW

File: client/src/pages/Messages.js [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+    this.state = {
+      conversations: [],
+      message: "",
+      messages: [],
+      conversationId: "",
+      recipientIds: [],
+      recipientProfiles: [],
+      open: false,
+      token: localStorage.getItem("jwtToken"),
+      userId: localStorage.getItem("userId")
```
Comment (ykurata) [inline client/src/pages/Messages.js]: I added this, but I don't use it, please remove this line. ( userId)

File: client/src/pages/Messages.js [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+
+  render() {
+    const { classes } = this.props;
+    const message = this.state.messages.map((message, i) => (
+      <p key={i} className={classes.sentMessageLength}>
+        <span className={classes.test}>{message}</span>
+      </p>
+    ));
+
+    const converId = this.state.conversations.map((con, i) => (
```
Comment (ykurata) [inline client/src/pages/Messages.js]: I create this coverId button for test purpose, so please remove these lines!

File: client/src/pages/Messages.js [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+                      <p id="transition-modal-description">
+                        react-transition-group animates me.
+                      </p>
+                    </div>
+                  </Fade>
+                </Modal>
+              </Grid>
+              <Grid item xs={12}>
+                <Card className={classes.cardStyle}>
+                  {converId}
```
Comment (ykurata) [inline client/src/pages/Messages.js]: Please remove {converId}

**Review by CometS1 — APPROVED** [review]

**Review by ykurata — APPROVED** [review]
Looks good! Please move buttons I created

**Pull request merged**